internal/validate/validate.go

```diff
@@ -158,3 +160,23 @@ func DBUsername(val interface{}) error {
 
 	return fmt.Errorf("%w: %s", ErrInvalidDBUsername, name)
 }
+
+var ErrWeakPassword = errors.New("the password provided is too common")
+var ErrShortPassword = errors.New("the password provided is too short [Minimum 10 characters]")
+
+func WeakPassword(val interface{}) error {
+	password, ok := val.(string)
+	if !ok {
+		return ErrWeakPassword
+	}
+
+	if commonPasswords[password] {
+		return ErrWeakPassword
+	}
+
+	if len(password) < minPasswordLength {
+		return ErrShortPassword
```

Comment:
based on previous comment on formatting
```suggestion
		return fmt.Erorf("%w min: %d", ErrShortPassword, minPasswordLength)
```